core/src/test/scala/integration/kafka/api/QuotasTest.scala

```diff
@@ -156,7 +172,7 @@ class QuotasTest extends KafkaServerTestHarness {
                                             RequestKeys.nameForKey(RequestKeys.ProduceKey),
                                             "Tracking throttle-time per client",
                                             "client-id", producerId2)
-    Assert.assertEquals("Should not have been throttled", 0.0, allMetrics(producerMetricName).value())
+    assertEquals("Should not have been throttled", 0.0, allMetrics(producerMetricName).value(), EPS)
```

Comment:
If the client is not throttled, the throttle-time should always be 0, right? Is there a need to for using EPS? Ditto below.